OnShift/SendBird-JavaScript
# Pl 36 utilize sendbird token system

# Things Done
- Pass in token to connect
- Remove unused "CallBack" function
- 

# How To Test
- Start up the app, within engage or Kangaroo
- If it loads, you have a good token
- Check console statements for errors like `{name: "SendBirdException", code: 400302, message: "Access token is not valid."}`

# Related PRs
